src/DivanteClipboardBundle.php

```diff
@@ -17,6 +17,12 @@
  */
 class DivanteClipboardBundle extends AbstractPimcoreBundle
 {
+    protected $name = "Pimcore 5 Clipboard";
```

Comment:
IMHO variables ARE NOT needed here:
* they won't change, so instead of variables it should be constants
* but even constants will break YAGNI rule - **so IMHO finally it should be directly result of `getDescription()`, `getVersion()` and `getName()` methods

Reply:
Agree. Maybe later we'll change it into getting version from composer.

Reply:
I'm not sure but leaving `$name` variable is not the best idea also... 

`$name` is a Symfony variable from `\Symfony\Component\HttpKernel\Bundle\Bundle`. I'm quite sure (70%) that setting name here will break some functionalities (e.g. see `\Symfony\Component\HttpKernel\Bundle\Bundle::getContainerExtensionClass`) 

IMHO instead of this you SHOULD override `getNiceName` method and there add the `nice name` 😄